homeassistant/components/arve/coordinator.py

```diff
@@ -0,0 +1,60 @@
+"""Coordinator for the Arve integration."""
+
+from __future__ import annotations
+
+from datetime import timedelta
+
+from asyncarve import Arve, ArveConnectionError, ArveError, ArveSensProData
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_ACCESS_TOKEN, CONF_CLIENT_SECRET, CONF_NAME
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryError
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN, LOGGER
+
+
+class ArveCoordinator(DataUpdateCoordinator):
```

Comment:
```suggestion
class ArveCoordinator(DataUpdateCoordinator[..]):
```
Please type the update coordinator the return type of `_async_update_data`